docs/markdown/Continuous-Integration.md

```diff
@@ -44,6 +44,65 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then SDKROOT=$(xcodebuild -version -sdk macosx Path) meson builddir && ninja -C builddir test; fi
 ```
 
+## CircleCi for Linux (with Docker)
+
+For CI on Linux, [CircleCi](https://circleci.com/) is probably
+your best bet. Here's a sample `yml` file for use with that.
```

Comment:
I don't think this is true. I'm sure CircleCi works, but so does any other of the common CI providers that offer linux.

Reply:
This most definitely works, please check out my dummy/templates for your self.

[D](https://github.com/michaelbadcrumble/d-project)

Reply:
However I think you probably meant the words being marked up in green.  So I would be happy to rewrite that as soon as possible.

Reply:
Done.

Reply:
Thanks for changing the wording. But i still feel "fantastic" is not what the official meson documentation should say about a particular commercial CI company without a solid foundation for such a wording. I'm happy to have a CircleCi example, but i don't think the meson documentation should be using such wording here.

Reply:
I agree. Comments about external service providers should be neutral.

Reply:
It should go more along the lines of "if you wish to use CircleCI, then <rest of the text>".

Reply:
Ok made some changes and hopefully it works for the documentation.

Reply:
However as a suggestion I think there should be a guideline for how the documentation should be written.

Reply:
😕 not sure what happened but I know it could not have been the documentation change.

Reply:
I'd say use your best guess and don't worry too much when that does not line up with what others expect in the end and you have to reword something.

I think review of actual documentation works a lot better than anyone trying to come up with a complicated guide.